docs/docs/guides/evaluation/scorers.md

```diff
@@ -224,7 +224,7 @@ In Weave, Scorers are used to evaluate AI outputs and return evaluation metrics.
     ```
 
     ### Mapping Column Names with `columnMapping`
-    :::warning
+    :::important
 
     In TypeScript, this feature is currently on the `Evaluation` object, not individual scorers!
```

Comment:
Do you want to remove this exclamation point too?

Reply:
Good call, removed in https://github.com/wandb/weave/pull/3253/commits/ddb8af965d4b410474856e6d75c2be8c9abe257f